WORKSPACE

```diff
@@ -33,11 +33,11 @@ bind(
 )
 
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
-# envoy-wasm commit date: 08/26/2019
+# envoy-wasm commit date: 09/12/2019
```

Comment:
Could you please update the SHA to the anyone after 09/24/2019, so that it picks the SDS change?

Reply:
Sure, done.

Reply:
Thanks a lot!